site/docs/plugins/commands.md

```diff
@@ -200,25 +200,26 @@ Let's say we want to have developer-only commands.
 We can achieve that with the following code structure:
 
 ```ascii
-src/
-├─ commands/
-│  ├─ admin.ts
-│  ├─ users/
-│  │  ├─ group.ts
-│  │  ├─ say-hi.ts
-│  │  ├─ say-bye.ts
-│  │  ├─ ...
-├─ bot.ts
-├─ types.ts
-tsconfig.json
+.
+├── types.ts
+├── bot.ts
+└── commands/
+    ├── admin.ts
+    └── users/
+        ├── group.ts
+        ├── say-hello.ts
+        └── say-bye.ts
 ```
 
-The following code group exemplifies how we could implement a developer only command group, and update the Telegram client Command menu accordingly.
-Make sure you take notice of the different patterns being use in the `admin.ts` and `group.ts` file-tabs.
+The following code group exemplifies how we could implement a developer only command group, and update the Telegram client command menu accordingly.
+Make sure you take notice of the different patterns being used in the `admin.ts` and `group.ts` file-tabs.
 
 ::: code-group
 
 ```ts [types.ts]
+import type { Context } from "grammy";
+import type { CommandsFlavor } from "grammy_commands";
```

Comment:
we've already showed how to import this, and there's no `grammy_commands` module. for Deno, it should be the entire URL, and for node it should be `@grammyjs/commands`. that's why we detailed everything in the specific session, so we wouldne't have to keep repeating the imports for each runtime.

```suggestion
```